core/src/test/java/org/apache/brooklyn/core/catalog/internal/SpecParameterInMetaTest.java

```diff
@@ -43,18 +49,40 @@
 public class SpecParameterInMetaTest {
     private ManagementContext mgmt;
     private BrooklynCatalog catalog;
-    private String spec;
+    private String specId;
 
     @BeforeMethod(alwaysRun=true)
     public void setUp() {
         mgmt = LocalManagementContextForTests.newInstanceWithOsgi();
         catalog = mgmt.getCatalog();
-        spec = TestToSpecTransformer.registerSpec(EntitySpec.create(BasicEntity.class));
+        StaticTypePlanTransformer.forceInstall();
+        PlanToSpecFactory.forceAvailable(TestToSpecTransformer.class, JavaCatalogToSpecTransformer.class);
```

Comment:
Part of the tests depend on StaticTypePlanTransformer, other part on PlanToSpecFactory, but not at the same time? If so it would be nice to split them in separate files so the dependencies are clear.